src/const.ts

```diff
@@ -226,6 +228,24 @@ export const DISABLED_TOKEN_MAPS = Object.keys(disabledTokens).reduce<DisabledTo
   },
 )
 
+type SupportedChainId = typeof ALL_SUPPORTED_CHAIN_IDS[Network]
```

Comment:
I'm thinking out loud, but shouldn't we replace the `Network` enum for the `ALL_SUPPORTED_CHAIN_IDS` type throughout the app? Currently the `Network` enum has lots of networks that are not used nor referenced (I can be wrong, I just did a quick search), so it's possible to do something like this:

```
import { SupportedChainId } from '@cowprotocol/cow-sdk/dist/constants/chains' // it's a boomer we haven't exposed SupportedChainId in the root export
export import Network = SupportedChainId
```

Then we need to rename the places where Network was used to match the network names

Reply:
This throws Module not found error. It would be better to do this once `SupportedChainId` is exported as a type.